projects/canopy/src/lib/table/table.notes.ts

```diff
@@ -46,29 +76,44 @@ and in your HTML:
         </lg-table-expanded-detail>
       </td>
     </tr>
+  </tbody>
+</table>
+~~~
+
+### Table with long copy
+
+Sometimes a table can be used to display large amounts of copy, including buttons, links and headings, such as this example which explains the benefits of a policy. Note that each \`\`LgTableCellComponent\`\` has the \`\`stackSm\`\` Input set to \`\`true\`\`, which stacks the label and content on top of each other on small devices, instead of side by side. Also note the use of \`\`<colgroup>\`\` to control width of the columns on large devices. This can be seen on the <a href="/story/components-table--with-long-copy">Table With Long Copy story</a>.
+
+~~~html
+<table lg-table>
+  <colgroup>
+    <col span="1" style="width: 65%;" />
+    <col span="1" style="width: 35%;" />
+  </colgroup>
+  <thead lg-table-head>
     <tr lg-table-row>
-      <td>
-        <lg-table-row-toggle
-        (click)="<function-to-handle-click>"
-        [isActive]="<logic-to-handle-toggle>"
-        >
-        </lg-table-row-toggle>
-      </td>
-      <td lg-table-cell>Albert Camus</td>
-      <td lg-table-cell>The Plague</td>
-      <td lg-table-cell>1947</td>
+      <th lg-table-head-cell>Benefit</th>
+      <th lg-table-head-cell>Who receives this money?</th>
     </tr>
-    <tr lg-table-row [isHidden]="<logic-to-handle-row>">
-      <td lg-table-cell>
-        <lg-table-expanded-detail>
-          Detail content goes here
-        </lg-table-expanded-detail>
+  </thead>
+
+  <tbody lg-table-body>
+    <tr lg-table-row>
+      <td lg-table-cell [stackSm]="true">
+        <h3>Benefit one: Dependant person benefit</h3>
+        <p>You have chosen a dependant person to receive your income when you die. This person is Joe Bloggs. If this person has passed away, is no longer your spouse or civil partner or no longer financially dependant on you at the date of your death, then we will not pay out any income. Please refer to your policy document for more details.</p>
```

Comment:
Cool, I'm happy to use Lorum Ipsum. I remember when we had the original Canopy repo, we were avoiding anything L&G specific before we went open source, but I don't think we've come to a conclusion about whether to use real content or not since. I'll change this one and then open a discussion topic

Reply:
Done ✅